### DISCUSSION TITLE
iperf 3.1.5 not producing server summary for UDP

### DISCUSSION THREAD

**Discussion by quite68** [esnet/iperf#1248] [Q&A]
Server running on Android:
iperf3-arm   --server   --bind 192.168.1.6 --port 5001 --interval 1 -f k

Summary is 0
```
[  6]   9.00-10.00  sec  5.51 MBytes  46225 Kbits/sec  0.071 ms  250/4239 (5.9%)  
[  6]  10.00-10.02  sec   105 KBytes  36572 Kbits/sec  0.112 ms  0/74 (0%)  
- - - - - - - - - - - - - - - - - - - - - - - - -
[ ID] Interval           Transfer     Bandwidth       Jitter    Lost/Total Datagrams
[  6]   0.00-10.02  sec  0.00 Bytes  0.00 Kbits/sec  0.112 ms  2117/42753 (5%)  
```

Client running on Linux:
iperf3-x86  --client 192.168.1.6 --time 10 --bind 192.168.1.8 --port 5001 --interval 1 --parallel 1 --udp --bandwidth 50M -f k

Shows good summary:
```
[  4]   8.00-9.00   sec  5.96 MBytes  49976 Kbits/sec  4315  
[  4]   9.00-10.00  sec  5.96 MBytes  49972 Kbits/sec  4313  
- - - - - - - - - - - - - - - - - - - - - - - - -
[ ID] Interval           Transfer     Bandwidth       Jitter    Lost/Total Datagrams
[  4]   0.00-10.00  sec  59.0 MBytes  49524 Kbits/sec  0.112 ms  2117/42753 (5%)  
[  4] Sent 42753 datagrams
```

Any idea why the server summary is 0?

**Comment by bmah888** [comment]
I recommend that you update to a newer version of iperf3 on the server if you can (3.10.1 is current as of this writing). We actually fixed a bunch of UDP-related bugs in 3.2 (including some related to computing and printing statistics). I think if you can get to that version (or newer) it'll work better.
